src/renderers/native/ReactNativeAttributePayload.js

```diff
@@ -314,6 +318,9 @@ function diffProperties(
   for (var propKey in nextProps) {
     attributeConfig = validAttributes[propKey];
     if (!attributeConfig) {
+      if (__DEV__ && enableStrictAttributeValidation === true && propKey !== 'children') {
+        throw new Error('unsupported attribute: ' + propKey);
```

Comment:
Would it make more sense to warn instead of throw? If it's a noop currently it doesn't make sense to start throwing errors. If so you should use the `warning` package. Even if it's decided this would throw you would want to use the `invariant` package here instead of throwing the error manually.